### PR TITLE
Country profile - datasets for overview page

### PR DESCRIPTION
**Pull request by spoonerf**

### CODE REVIEW

**Comment by spoonerf** [comment]
The data behind the charts on this example page - https://tufte-owid.netlify.app/italy

**Comment by spoonerf** [comment]
Hey @pabloarosado and @lucasrodes! 

This is a bit of an experiment for creating a download for the country profile overview pages e.g. https://tufte-owid.netlify.app/italy. The idea is to create a download of the same variables for each country, where that data is available. I guess one big thing that is missing are tables for regions, but I just wanted to see if this is something that could work.

**Review by pabloarosado — APPROVED** [review]
All good, I made some suggestions (for now or for future steps), feel free to ignore them.

File: etl/steps/data/garden/country_profile/2022/overview.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+        # assert variables are in the table - if not throw an error
+        # fix the index to be (year, entity_name)
+        t = t.reset_index().drop(columns=["entity_id", "entity_code"]).set_index(["entity_name", "year"])[variables]
+
+        if t_all.shape == tuple([0, 0]):
+            # first time around
+            t_all = t
+        else:
+            t_all = t_all.join(t, how="outer")  # omg hope
```
[the remark this answers is not in the document]
Comment (spoonerf) [reply etl/steps/data/garden/country_profile/2022/overview.py:209]: Haha - lars wrote this bit and added that comment 😆

**Pull request merged**